src/PyCall.jl

```diff
@@ -291,8 +308,29 @@ function getindex(o::PyObject, s::AbstractString)
     return PyObject(p)
 end
 
-getindex(o::PyObject, s::Symbol) = convert(PyAny, getindex(o, string(s)))
+getproperty(o::PyObject, s::Symbol) = convert(PyAny, getproperty(o, string(s)))
```

Comment:
If `s === :o`, it should call `PyPtr(o)` with a deprecation warning.

Reply:
Actually, it's probably fine to leave out this deprecation.  I did a quick audit of all of the registered packages using PyCall, and it looks like only PyPlot was accessing the internal `o` field of `PyObject`, and PyPlot has been fixed.